examples/modules/TMCM1638/TMCL/encoder_position_mode.py

```diff
@@ -0,0 +1,68 @@
+import pytrinamic
+from pytrinamic.connections import ConnectionManager
+from pytrinamic.modules import TMCM1638
+import time
+
+pytrinamic.show_info()
+# connection_manager = ConnectionManager("--interface serial_tmcl --port COM4 --data-rate 115200")
+connection_manager = ConnectionManager("--interface kvaser_tmcl --module-id 1")
+
+with connection_manager.connect() as my_interface:
+    module = TMCM1638(my_interface)
+    motor = module.motors[0]
+
+    # Define motor configuration for the TMCM-1638.
+    #
+    # The configuration is based on our standard BLDC motor (QBL4208-61-04-013-1024-AT).
+    # If you use a different motor be sure you have the right configuration setup otherwise the script may not work.
+
+    # drive configuration
+    motor.drive_settings.motor_type = motor.ENUM.MOTOR_TYPE_THREE_PHASE_BLDC
+    motor.drive_settings.pole_pairs = 4
+    motor.drive_settings.max_current = 2000
+    motor.drive_settings.commutation_mode = motor.ENUM.COMM_MODE_ABN_ENCODER
+    motor.drive_settings.target_reached_distance = 5
+    motor.drive_settings.target_reached_velocity = 500
+    motor.drive_settings.open_loop_current = 1000 
+    print(motor.drive_settings)
+
+    # encoder configuration 
+    motor.abn_encoder.resolution = 4096  # 16384
+    motor.abn_encoder.direction = 1
+    motor.abn_encoder.init_mode = motor.ENUM.ENCODER_INIT_MODE_0
+    print(motor.abn_encoder)
+
+    # motion settings 
+    motor.linear_ramp.max_velocity = 2000
+    motor.linear_ramp.max_acceleration = 1000
+    motor.linear_ramp.enabled = 1 
+    print(motor.linear_ramp)
+
+    motor.set_axis_parameter(motor.AP.PositionScaler, motor.abn_encoder.resolution)
+
+    # PI configuration 
+    motor.pid.torque_p = 500
+    motor.pid.torque_i = 1000
+    motor.pid.velocity_p = 500
+    motor.pid.velocity_i = 100
+    motor.pid.position_p = 300
+    print(motor.pid)
+
+    time.sleep(1.0)
+
+    # clear actual position 
+    motor.actual_position = 0
+
+    # set target position 
+    motor.move_to(motor.abn_encoder.resolution * 50)
+    while not(motor.get_position_reached()):
+        print("target position: " + str(motor.target_position) + " actual position: " + str(motor.actual_position))
```

Comment:
It would be best to use f-strings here, but we cannot use f-strings in our MicroPython. But we can at least use [str.format](https://docs.python.org/3/library/stdtypes.html#str.format) like this:
```python
print("target position: {} actual position: {}".format(motor.target_position, motor.actual_position))
```